code/modules/power/lighting.dm

```diff
@@ -722,16 +793,18 @@
 	if(istype(I, /obj/item/reagent_containers/syringe))
 		var/obj/item/reagent_containers/syringe/S = I
 
-		to_chat(user, "You inject the solution into the [src].")
-
-		if(S.reagents.has_reagent("plasma", 5) || S.reagents.has_reagent("plasma_dust", 5))
+		if(!length(S.reagents.reagent_list))
+			return
 
-			log_admin("LOG: [key_name(user)] injected a light with plasma, rigging it to explode.")
-			message_admins("LOG: [key_name_admin(user)] injected a light with plasma, rigging it to explode.")
+		else
```

Comment:
else is not required due to line 797

Reply:
That's a leftover from something else I tried, whoops!